Makefile

```diff
@@ -0,0 +1,35 @@
+build:
```

Comment:
please don't check in a Makefile into the repo.  If you want one locally that's fine if it's helpful for your workflow but we shouldn't put this into the codebase.